rust/start.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/zsh
+
+source ~/.zshrc
+
+export HYP_CHAINS_ARBITRUM_CUSTOMRPCURLS=$(rpcs mainnet3 arbitrum)
+export HYP_CHAINS_BASE_CUSTOMRPCURLS=$(rpcs mainnet3 base)
+export HYP_CHAINS_BSC_CUSTOMRPCURLS=$(rpcs mainnet3 bsc)
+export HYP_CHAINS_ETHEREUM_CUSTOMRPCURLS=$(rpcs mainnet3 ethereum)
+export HYP_CHAINS_OPTIMISM_CUSTOMRPCURLS=$(rpcs mainnet3 optimism)
```

Comment:
link to the `rpcs` rune in notion as well - since it's new and some people may confuse it with the other `rpc` rune